### PR TITLE
bump mariadb connector

### PR DESCRIPTION
**Pull request by ramonskie**
Jammy has a new version of cmake 
and 3.1.11 had a bug which resulted in this error
```
Building C object CMakeFiles/cmTC_ec300.dir/CheckFunctionExists.c.o
/usr/bin/cc   -Wunused -Wlogical-op -Wno-uninitialized -Wall -Wextra -Wformat-security -Wno-init-self -Wwrite-strings -Wshift-count-overflow -Wdeclaration-after-statement -Wno-undef -Wno-unknown-pragmas -DCHECK_FUNCTION_EXISTS=floor -o CMakeFiles/cmTC_ec300.dir/CheckFunctionExists.c.o -c /usr/share/cmake-3.22/Modules/CheckFunctionExists.c
<command-line>: warning: conflicting types for built-in function ‘floor’; expected ‘double(double)’ [-Wbuiltin-declaration-mismatch]
/usr/share/cmake-3.22/Modules/CheckFunctionExists.c:7:3: note: in expansion of macro ‘CHECK_FUNCTION_EXISTS’
    7 |   CHECK_FUNCTION_EXISTS(void);
      |   ^~~~~~~~~~~~~~~~~~~~~
/usr/share/cmake-3.22/Modules/CheckFunctionExists.c:1:1: note: ‘floor’ is declared in header ‘<math.h>’
  +++ |+#include <math.h>
    1 | #ifdef CHECK_FUNCTION_EXISTS
Linking C executable cmTC_ec300
/usr/bin/cmake -E cmake_link_script CMakeFiles/cmTC_ec300.dir/link.txt --verbose=1
/usr/bin/cc  -Wunused -Wlogical-op -Wno-uninitialized -Wall -Wextra -Wformat-security -Wno-init-self -Wwrite-strings -Wshift-count-overflow -Wdeclaration-after-statement -Wno-undef -Wno-unknown-pragmas -DCHECK_FUNCTION_EXISTS=floor -rdynamic CMakeFiles/cmTC_ec300.dir/CheckFunctionExists.c.o -o cmTC_ec300 
/usr/bin/ld: CMakeFiles/cmTC_ec300.dir/CheckFunctionExists.c.o: in function `main':
CheckFunctionExists.c:(.text+0x14): undefined reference to `floor'
collect2: error: ld returned 1 exit status
gmake[1]: *** [CMakeFiles/cmTC_ec300.dir/build.make:99: cmTC_ec300] Error 1
gmake[1]: Leaving directory '/var/vcap/bosh_ssh/bosh_03a2782828dd45d/mariadb-connector-c-3.1.11-src/bld/CMakeFiles/CMakeTmp'
gmake: *** [Makefile:127: cmTC_ec300/fast] Error 2
```
see release notes 
- https://mariadb.com/kb/en/mariadb-connector-c-3112-release-notes/
- https://mariadb.com/kb/en/mariadb-connector-c-3113-release-notes/

a version bumped fixed this issue locally

### CODE REVIEW

**Review by nouseforaname — APPROVED** [review]
lgtm

File: config/blobs.yml [hunk 1]
```diff
@@ -14,6 +14,10 @@ mysql/mariadb-connector-c-3.1.11-src.tar.gz:
   size: 752342
   object_id: 0939120b-af11-4faf-7cab-8609bc40d447
   sha: sha256:3e6f6c399493fe90efdc21a3fe70c30434b7480e8195642a959f1dd7a0fa5b0f
+mysql/mariadb-connector-c-3.1.13-src.tar.gz:
+  size: 968401
+  object_id: b3c4867c-ca9f-4819-4e11-e62595e7bd77
+  sha: sha256:0271a5edfd64b13bca5937267474e4747d832ec62e169fc2589d2ead63746875
```
Comment (rkoster) [inline config/blobs.yml:20]: Could you also remove the older version from this file

**Review by rkoster — CHANGES_REQUESTED** [review]
Just a small nitpick, rest looks good to me

File: config/blobs.yml (hunk 1 above)
Comment (ramonskie) [reply config/blobs.yml:16]: done

**Review by rkoster — APPROVED** [review]
LGTM

**Comment by rkoster** [comment]
Thanks @ramonskie

**Pull request merged**